### PR TITLE
[new release] ctypes_stubs_js (0.1)

### PR DESCRIPTION
**Pull request by hhugo**
Js_of_ocaml Javascript stubs for the OCaml ctypes library

- Project page: <a href="https://gitlab.com/nomadic-labs/ctypes_stubs_js">https://gitlab.com/nomadic-labs/ctypes_stubs_js</a>

## Features/Changes
* Initial release

### CODE REVIEW

File: packages/ctypes_stubs_js/ctypes_stubs_js.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+license: "MIT"
+bug-reports: "https://gitlab.com/nomadic-labs/ctypes_stubs_js/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/ctypes_stubs_js.git"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "integers_stubs_js"
+  "dune" {build & >= "2.9"}
```
Comment (mseri) [inline packages/ctypes_stubs_js/ctypes_stubs_js.0.1/opam:16]:
```suggestion
  "dune" {>= "2.9"}
```

**Comment by hhugo** [comment]
I've updated the hash

**Comment by hhugo** [comment]
I think this is ready to be merged.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**